website/source/intro/getting-started/build-image.html.md

```diff
@@ -200,4 +198,269 @@ image was pretty useless in this case (nothing was changed about it), this page
+            "ssh_username": "ubuntu",
+            "type": "amazon-ebs"
+        }
+    ],
+    "provisioners": [
+        {
+            "type": "file",
+            "source": "./welcome.txt",
+            "destination": "/home/ubuntu/"
+        },
+        {
+            "type": "shell",
+            "inline":["ls -al /home/ubuntu",
+                      "cat /home/ubuntu/welcome.txt"]
+        },
+        {
+            "type": "shell",
+            "script": "./example.sh"
+        }
+    ]
+}
+```
+
+and to build, run `packer build firstrun.json`
+
+Note that this example provides a `source_ami` instead of a `source_ami_filter` -- this means the example may be out of date by the time you try to use it, but it is provided here so you can see what it looks like to use an ami by name.
+
+Your output will look like this: 
+
+```
+amazon-ebs output will be in this color.
+
+==> amazon-ebs: Prevalidating AMI Name: packer-linux-aws-demo-1507231105
+    amazon-ebs: Found Image ID: ami-fce3c696
+==> amazon-ebs: Creating temporary keypair: packer_59d68581-e3e6-eb35-4ae3-c98d55cfa04f
+==> amazon-ebs: Creating temporary security group for this instance: packer_59d68584-cf8a-d0af-ad82-e058593945ea
+==> amazon-ebs: Authorizing access to port 22 on the temporary security group...
+==> amazon-ebs: Launching a source AWS instance...
+==> amazon-ebs: Adding tags to source instance
+    amazon-ebs: Adding tag: "Name": "Packer Builder"
+    amazon-ebs: Instance ID: i-013e8fb2ced4d714c
+==> amazon-ebs: Waiting for instance (i-013e8fb2ced4d714c) to become ready...
+==> amazon-ebs: Waiting for SSH to become available...
+==> amazon-ebs: Connected to SSH!
+==> amazon-ebs: Uploading ./scripts/welcome.txt => /home/ubuntu/
+==> amazon-ebs: Provisioning with shell script: /var/folders/8t/0yb5q0_x6mb2jldqq_vjn3lr0000gn/T/packer-shell661094204
+    amazon-ebs: total 32
+    amazon-ebs: drwxr-xr-x 4 ubuntu ubuntu 4096 Oct  5 19:19 .
+    amazon-ebs: drwxr-xr-x 3 root   root   4096 Oct  5 19:19 ..
+    amazon-ebs: -rw-r--r-- 1 ubuntu ubuntu  220 Apr  9  2014 .bash_logout
+    amazon-ebs: -rw-r--r-- 1 ubuntu ubuntu 3637 Apr  9  2014 .bashrc
+    amazon-ebs: drwx------ 2 ubuntu ubuntu 4096 Oct  5 19:19 .cache
+    amazon-ebs: -rw-r--r-- 1 ubuntu ubuntu  675 Apr  9  2014 .profile
+    amazon-ebs: drwx------ 2 ubuntu ubuntu 4096 Oct  5 19:19 .ssh
+    amazon-ebs: -rw-r--r-- 1 ubuntu ubuntu   18 Oct  5 19:19 welcome.txt
+    amazon-ebs: WELCOME TO PACKER!
+==> amazon-ebs: Provisioning with shell script: ./example.sh
+    amazon-ebs: hello
+==> amazon-ebs: Stopping the source instance...
+    amazon-ebs: Stopping instance, attempt 1
+==> amazon-ebs: Waiting for the instance to stop...
+==> amazon-ebs: Creating the AMI: packer-linux-aws-demo-1507231105
+    amazon-ebs: AMI: ami-f76ea98d
+==> amazon-ebs: Waiting for AMI to become ready...
+```
+
+### A windows example
+
+Note that this uses a larger instance.  You will be charged for it.
```

Comment:
is it not possible to run on a smaller instance? Should also point out that using windows AMIs in general incurs a fee

Reply:
I tried, and it hangs.  I'll call out the windows fee too.